packages/common/src/core/schemas/internal/metrics/interfaces.ts

```diff
@@ -0,0 +1,62 @@
+export enum ALIGNMENTS {
+  start = "start",
+  center = "center",
+  end = "end",
+}
+export enum CORNERS {
```

Comment:
Makes sense...I think we can move them up into a generic types file.

I feel like `/schemas` might be a good place for them...I would see us using them in the `schema` if anything, listing properties as enums -- we'll also probably use them in the actual ui (like in the `arcgis-stat-card` component), but who knows what other components in the ui will end up using them...so having them in a central place feels necessary. 

Since we need them to be centralized, and we know we'll share them in the schema before using them in each component as well, I feel like there's a somewhat solid case for them to live in `schemas/types.ts`. Thoughts?